# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieParser.h
# Commit: d10ae1bb47a8
# Full Hash: d10ae1bb47a8b1cc81718ace05688e97d583e27f
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-10 21:44:02
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 5: Exempt third-party cookie blocking if the channel is on the exception list. r=cookie-reviewers,bvandersloot,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223384
# ==============================================================================

diff -r 336214ad7891 -r d10ae1bb47a8 netwerk/cookie/CookieParser.h
--- a/netwerk/cookie/CookieParser.h	Fri Jan 10 08:24:29 2025 +0000
+++ b/netwerk/cookie/CookieParser.h	Fri Jan 10 08:24:30 2025 +0000
@@ -50,7 +50,7 @@
              CookieStatus aStatus, nsCString& aCookieHeader,
              const nsACString& aDateHeader, bool aFromHttp,
              bool aIsForeignAndNotAddon, bool aPartitionedOnly,
-             bool aIsInPrivateBrowsing);
+             bool aIsInPrivateBrowsing, bool aOn3pcbException);
 
   bool ContainsCookie() const {
     MOZ_ASSERT_IF(mContainsCookie, mRejection == NoRejection);